.github/ISSUE_TEMPLATE/new-blog-post.md

```diff
@@ -12,6 +12,15 @@ Issue Title: [PROPOSAL] Blog post title
 * Categories:
 * Deadlines: are there any deadlines for this? do we need to wait for any other event?
 
+## Blog process
+
+- Create a proposal on this blog repo
+- Get proposal accepted by DevRel
+- Create a copy of [this gdoc](https://docs.google.com/document/d/1BxhHlkxytm-xCDVy4vRkdyjqwe9w7pIWoauaLpMMreE/edit#heading=h.rujpt9lvqbj8) & update the template header 
+- Add the draft of you blog post directly in gdoc with grammarly enabled
```

Comment:
Should the draft be stored in any special directory?

Reply:
Yes in a google doc folder called Blog posts under DevRel

Reply:
```suggestion
- Add the draft of you blog post directly in gdoc with grammarly enabled, and store it under [DevRel>BlogPosts](https://drive.google.com/drive/folders/1PGIqQ284kAXWgY50Nco9lRF6VIYGyxKg)
```